geomesa-utils/src/test/scala/org/locationtech/geomesa/utils/geotools/GeometryUtilsTest.scala

```diff
@@ -37,5 +38,36 @@ class GeometryUtilsTest extends Specification {
         min must beLessThanOrEqualTo(max)
       }
     }
+
+    "split bounding box envelopes" >> {
+      val unchanged = Seq(
+        new ReferencedEnvelope(-180, 180, -90, 90, CRS_EPSG_4326),
```

Comment:
What happens if a bbox covers and contains this one?  E.g., if the bbox is larger than the world?

Reply:
thanks, added a check and test for that.